app/src/ui/banners/banner.tsx

```diff
@@ -4,49 +4,64 @@ import * as OcticonSymbol from '../octicons/octicons.generated'
 
 interface IBannerProps {
   readonly id?: string
-  readonly timeout?: number
```

Comment:
Sorry.. I maybe have been confusing when talking about this in the team sync, but I do not think we can completely do away with this as right now our current users have muscle memory around these automatically disappearing.

If we are going to remove them, we also need to add in the logic that dismisses them when they lose focus.

Otherwise, I think it is just prudent to elongate the timer on the banners that have undo buttons.